compose.yaml

```diff
@@ -1,5 +1,3 @@
-version: '3.8'
```

Comment:
why we deleting version?

Reply:
It's actually deprecated that's why you will be getting warning but it's just a warning.